backend/actions.py

```diff
@@ -147,18 +146,33 @@ def hankering(self):
 
   def apply_effect(self):
     for target in self.targets:
-      target.loyalty += copysign(self.heal, self.subject.loyalty)
+      target.heal(self.heal * self.subject.level)
 
   def get_info(self):
     return {**super().get_info(),
             'heal': self.heal}
 
 
+class EdibleWildlife(Action):
+  def apply_effect(self):
+    self.subject.heal(0.01 * self.subject.influence)
+
+class SafetyCollar(Action):
+  inspiration = 3
+  def hankering(self):
+    return 4 if self.target else 0
+  def prepare(self, state):
+    self.target = self.subject.find_closest_opponent(state)
+  def apply_effect(self):
+    if target:
+      target.status.append({'type': 'SafetyCollar', 'damage': self.subject.influence * 5})
+
+
 class ComeToPapa(Action):
   default_hankering = 10
   pull_range = 1
   cooldown = 5
-  inspiration = 1
+  inspiration = 3
```

Comment:
If there's an inspiration=1 action, the user barely even sees the inspiration. We get it and immediately spend it. I think it's more interesting to require 3 or 5 as the default.